binaryaudit/db.py

```diff
@@ -118,3 +122,89 @@ def get_product_id(self, productname, derivativename) -> int:
         product_id = record.ProductID
 
         return product_id
+
+    def insert_main_transaction(self, build_id, product_id, buildurl="", logurl="") -> None:
+        '''
+        inserts new object to the [main table]
+        '''
+        session = self._acquire_session()
+        new_tbl_entry = self.binaryaudit_transaction_main_tbl(
+                        BuildID=build_id, 
+                        ProductID=product_id, 
+                        BuildUrl=buildurl, 
+                        LogUrl=logurl, 
+                        Result='pending'
```

Comment:
Perhaps a good idea for later would be to formalize the result value.

thanks.